EIPS/eip-5005.md

```diff
@@ -0,0 +1,225 @@
+---
+eip: 5005
+title: Zodiac
+description: A composable design philosophy for DAOs.
+author: Auryn Macmillan (@auryn-macmillan), Kei Kreutler (@keikreutler)
+discussions-to: https://ethereum-magicians.org/t/eip-zodiac-a-composable-design-philosophy-for-daos/8963
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-14
+requires: 165
+---
+
+## Abstract
+Zodiac is a philosophy and open standard for composable and interoperable DAO tooling. The key insight is that by separating account (thing that holds tokens, controls systems, and is referenced by others) from the logic that controls it, and making the account agnostic to the logic that controls it, we can create a DAO ecosystem that is much more composable and interoperable, along with being compatible with the vast majority of existing DAO tooling. In short, Zodiac encourages DAOs to use an "avatar" contract (like the Gnosis Safe) as their account, and to then control that avatar with just about any combination of DAO tools and frameworks (Aragon, Colony, Compound/OZ Governor, DAOStack, Moloch, Orca, Tribute, etc).
+
+## Motivation
+Currently, most DAO tools and frameworks are built as somewhat monolithic systems. Wherein account and control logic are coupled, either in the same contract or in a tightly bound system of contracts. This needlessly inhibits the future flexibility of organizations using these tools and encourages platform lock-in via extraordinarily high switching costs.
+
+By using the Zodiac standard to decouple account and control logic, organizations are able to:
+1. Enable flexible, module-based control of programmable accounts
+2. Easily switch between frameworks without unnecessary overhead.
+3. Enable multiple control mechanism in parallel.
+4. Enable cross-chain / cross-layer governance.
+5. Progressively decentralize their governance as their project and community matures.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+The Zodiac standard consists of four key concepts Avatars, Modules, Modifiers, and Guards:
+1. **Avatars** are programmable Ethereum accounts, like the [Gnosis Safe](https://gnosis-safe.io). Avatars are the address that holds balances, owns systems, executes transaction, is referenced externally, and ultimately represents your DAO. Avatars MUST expose the `IAvatar` interface.
+2. **Modules** are contracts enabled by an Avatar that implement some control logic.
+3. **Modifiers** are contracts that sit between Modules and Avatars to modify the Module's behavior. For example, they might enforce a delay on all functions a Module attempts to execute or limit the scope of transactions that can be initiated by the module. Modifiers MUST expose the `IAvatar` interface.
```

Comment:
Is it `Module` or `module`? I'd look through and ensure you have consistent capitalization.

Reply:
Good catch. Will resolve.